File: qualifier-week-3/graded assignments/GrPA1 - Graded.c
#include <stdio.h>
#include <math.h>
int main() {
    double a,b,c,x1,x2;
    scanf("%lf%lf%lf", &a,&b,&c);
    // Write solution code below
    double discriminant;
    discriminant = b * b - 4 * a * c;
    x1 = (-b + sqrt(discriminant)) / (2 * a);
    x2 = (-b - sqrt(discriminant)) / (2 * a);
    printf("%.2lf %.2lf",x1,x2);
    return 0;
}

File: qualifier-week-3/graded assignments/GrPA1 - Graded.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var a, b, c, x1, x2 float64
	fmt.Scan(&a, &b, &c)
	x1 = (-b - (math.Sqrt(b*b - 4*a*c))) / 2 * a
	x2 = (-b + (math.Sqrt(b*b - 4*a*c))) / 2 * a
	fmt.Printf("%.2f, %.2f\n", x1, x2)
}
